Sources/Core/AWSClientRuntime/Auth/AuthSchemes/SigV4Util.swift

```diff
@@ -0,0 +1,35 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import ClientRuntime
+
+public class SigV4Util {
+    static let usesSignedBodyHeader = ["S3", "Glacier", "S3 Control"]
+    static let forceUnsignedBodyForPresigningURL = [
+        "S3": ["getObject", "putObject"]
+    ]
+
+    static func shouldForceUnsignedBody(flow: FlowType, serviceName: String, opName: String) -> Bool {
```

Comment:
Nit: Would like more comments on these Util functions, especially detailing what they do and what the input types are like:

```
/// Determines whether to force an unsigned body based on the service name, operation name, and flow type.
/// - Parameters:
///   - flow: The type of flow (e.g., presigning a URL).
///   - serviceName: The name of the service (e.g., "S3").
///   - opName: The name of the operation (e.g., "GetObject").
/// - Returns: A Boolean indicating whether to force an unsigned body.
static func shouldForceUnsignedBody(...)
```

Reply:
Changed as suggested.